storage/innobase/xtrabackup/test/t/backup_locks.sh

```diff
@@ -26,6 +26,9 @@ $MYSQL $MYSQL_ARGS -Ns -e \
 
 binlog_stmts=$lock_binlog_used
 
+# Binlog backups are now always performed
+((binlog_stmts+=1))
```

Comment:
can you explain why this is needed ?

Reply:
Since we are now backing up the binlogs, this is for the extra call to lock tables.